charts/graph-node/Chart.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v2
+name: graph-node
+description: Graph Node indexes data from blockchains such as Ethereum and serves it over GraphQL
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 1.0.1
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
+# It is recommended to use it with quotes.
+appVersion: "0.25.0"
+
+keywords:
+  - graph-node
+  - GraphQL
+  - Ethereum
+home: https://thegraph.com/
+sources:
+  - https://github.com/graphprotocol/graph-node
+
+dependencies:
+  - name: postgresql
+    version: 10.13.15
+    repository: https://charts.bitnami.com/bitnami/
+    condition: postgresql.enabled
+  - name: ipfs
+    version: 1.0.1
+    repository: "file://../ipfs"
```

Comment:
I suggest we use the remote repo here

Reply:
I will change when the charts are deployed and I can specify our repository, otherwise the pipeline will crash